tofu/geom/_comp.py

```diff
@@ -205,16 +184,16 @@ def get_paths_from_svg(
     # Derive usable data
     kstr = 'fill:'
     lk = list(dpath.keys())
+    ref = None
     for ii, k0 in enumerate(lk):
 
         v0 = dpath[k0]
-        dpath[k0]['poly'] = _get_pts_from_path_svg(
-            v0['poly'],
-            res=res,
-            r0=r0,
-            z0=z0,
-            scale=scale,
-        )
+        poly, isref = _get_pts_from_path_svg(v0['poly'], res=res)
+        if isref is True:
+            ref = poly
+            del dpath[k0]
+            continue
+        dpath[k0]['poly'] = poly
```

Comment:
if I understand correctly, you will check all paths and see if any are references, if so, you keep the poly to compute automatic scaling and delete from the configuration? And how do you know if there are? just because there is a "z" in the poly?

Reply:
Yes, in svg formatting, a ''z at the end of the str representation of a path means it is closed. If there is only one path with no z it is a unclosed path so it is the straight line